extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -165,8 +165,8 @@ export class ConsentPolicyManager {
     if (!ConsentPolicyManager.isMultiSupported(this.ampdoc_.win)) {
       // If customized policy is not supported
       if (!WHITELIST_POLICY[policyId]) {
-        user().error(TAG, `can not find defined policy ${policyId}, ` +
-          'only supported predefined policy supported now');
+        user().error(TAG, 'can not find defined policy %s, ' +
+          'only supported predefined policy supported now', policyId);
```

Comment:
nit: this is a bit hard to understand. How about 'only predefined policies are supported'?

Reply:
changed.